exercises/02-generate-odata-clients.md

```diff
@@ -0,0 +1,28 @@
+# Generate your own OData clients
+
+There are many scenarios where you have either your own OData service or want to integrate with another SAP OData service. For those cases, you can generate your own OData client as described in the following.
+
+## Download EDMX file
+
+Download the service specification file for [SAP S/4HANA Cloud Workforce Timesheet service](https://api.sap.com/api/API_MANAGE_WORKFORCE_TIMESHEET/overview) and [SAP SuccessFactors Employee Central Time Off service](https://api.sap.com/api/ECTimeOff/overview) from the SAP Business API Hub and put it into the [resources/service-specs](../resources/service-specs/).
```

Comment:
[q] should we put the service specs  also in the repo? Then we could not check in the generated clients but generate them if needed? @marikaner  opinions?